content/src/content/jcr_root/apps/core/wcm/components/page/v3/page/README.md

```diff
@@ -0,0 +1,151 @@
+7. `./subtitle` - defines the page subtitle.
+8. `./jcr:description` - defines the page description.
+9.  `./onTime` - defines when the page should become available.
+10. `./offTime` - defines when the page should no longer be available.
+11. `./sling:vanityPath` - defines a vanity URL at which the page could be accessed.
+12. `./sling:redirect` - if `true`, the page will redirect to its defined vanity URL.
+13. `./jcr:language` - defines the language set for this page, to help with `i18n`.
+14. `./cq:designPath` - defines where the design for the page is stored.
+15. `./sling:alias` - defines a Sling alias for the page, so that it can be accessed at a different resource path.
+16. `./cq:allowedTemplates` - defines a template or a list of templates that the page should use.
+17. `./cq:exportTemplate` - defines the template used for exporting the page for content synchronisation.
+18. `./socialMedia` - defines the enabled social media configurations.
+19. `./variantPath` - allows defining the social media variation experience fragment to be used for generating page metadata for social
+media
+20. `./cq:contextHubPath` - defines the Context Path configuration used by this page.
+21. `./cq:contextHubSegmentsPath` - defines the Context Path Segments Path.
+22. `./mainContentSelector` - defines the ID of the main content element of the page (used by the "skip to main content" accessibility feature).
+23. `./id` - defines the component HTML ID attribute.
+
+## Web Resources Client Library
+A web resources client library can be defined at the template level (see `./appResourcesClientlib` component policy configuration).
+This client library has to provide the following structure:
+
+```json
+{
+  "<client library folder>": {
+    "jcr:primaryType": "cq:ClientLibraryFolder",
+    "allowProxy": true,
+    "categories": [
+      "<category name>"
+    ],
+    "css.txt": {"jcr:primaryType": "nt:file"},
+    "resources": {
+      "jcr:primaryType": "nt:folder"
+      }
+  }
+}
+```
+
+The following files are expected in the `resources` folder, for maximum compatibility:
+
+Filename|Browser|Size
+--------|-------|----
+apple-touch-icon-180x180.png|Safari on iPhone| 180px x 180px
+apple-touch-icon-167x167.png|Safari on iPad Pro| 167px x 167px
+apple-touch-icon-152x152.png|Safari on iPad, iPad Mini| 152px x 152px
+apple-touch-icon-120x120.png|Safari on iPhone| 120px x 120px
+icon-192x192.png|Chrome, Opera|192px x 192px
+icon-310x310.png|Internet Explorer, Edge and Windows Phone|310px x 310px
+icon-310x150.png|Internet Explorer, Edge and Windows Phone|310px x 150px
+icon-150x150.png|Internet Explorer, Edge and Windows Phone|150px x 150px
+icon-70x70.png|Internet Explorer, Edge and Windows Phone|70px x 70px
+
+## Client Libraries
+The component provides a `core.wcm.components.page.v2.sharing` client library category that contains the JavaScript
```

Comment:
I would change `provides a` to `reuses the`, both here and at L143

Reply:
addressed in https://github.com/adobe/aem-core-wcm-components/pull/1429/commits/b350c0b58a66187eb6b0a7ab490e57d1d3f40640